examples/oidc/apisix/conf/apisix.yaml

```diff
@@ -11,6 +11,9 @@ routes:
     plugins:
       proxy-rewrite:
         uri: /jmap
+        headers:
+          set:
+            X-User-Secret: ${{X_USER_SECRET}}
```

Comment:
<3

Thank you so much for proposing this which is JUST a simple configuration change and to not require a new plugin release.

Please also make sure the clients have no way to pass `X-USER` headers

Reply:
> Please also make sure the clients have no way to pass X-USER headers

for basic auth, it was done by `jmap_service_basic_auth` 
for oidc auth, it was done by `RewriteXUserFilter` plugin